src/main.js

```diff
@@ -47,6 +47,9 @@ parser.addArgument(['dest'], {
 
 export default async function main(args=parser.parseArgs()) {
   try {
+    if (args.product.indexOf('emulator') != -1 && !args.fileSuffix) {
```

Comment:
Let's move this into `src/moz_build_info.js` and make `archiveFiletype` take an extra product argument.